python-tech-tour/1. Introduction to Python.ipynb

```diff
@@ -475,6 +477,88 @@
     "print('The work of Jared is', work[1])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Working with dictionaries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'Richard': 'Engineer', 'Jared': 'Operations', 'Dinesh': 'Engineer', 'Gilfoyle': 'Engineer', 'Erlich': 'Useless'}\n",
+      "length of the dictionary is 5\n",
+      "Gilfoyle is an Engineer\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "False"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "name_work = {\"Richard\" : \"Engineer\", \"Jared\" : \"Operations\", \"Dinesh\" : \"Engineer\", \"Gilfoyle\" : \"Engineer\", \"Erlich\" : \"Useless\"}\n",
```

Comment:
Split this into multiple lines.

Reply:
Okay, will update it with the json module.